src/main/java/com/infernostats/InfernoStatsPlugin.java

```diff
@@ -433,9 +447,13 @@ public void onChatMessage(ChatMessage event)
 			waveTimer.Pause();
 			removeWaveTimer();
 
-			wave.Finished(true);
+			wave.Finished(waveTimer.SplitTime(), true);
```

Comment:
It's been quite a long time since I've touched this code, but is there a reason for passing in waveTimer.splitTime() here? When Finished is called it sets its own stopTime, which is Instant.now() - the same time that waveTimer.SplitTime() _should_ represent.

Reply:
The SplitTime is the duration of the run so far while the stopTime is the current clock time - it didn't seem like it was possible to get the run time from that since the `startTime` is the start time of that wave, not the start of the run (otherwise yeah you could check stopTime - startTime)